tests/interfaces/test_single_interface.py

```diff
@@ -0,0 +1,103 @@
+import json
+from unittest.mock import patch
+from unittest import TestCase
+
+import responses
+from webapp.app import app
+from webapp.integrations.logic import Interfaces
+
+mock_interface_yaml = """
+name: test_interface
+
+version: 0
+status: published
+
+providers:
+  - name: ptest_provider
+    url: https://www.github.com/canonical/test_provider
+
+requirers:
+  - name: test-requirer
+    url: https://www.github.com/canonical/test_requirer
+
+"""
+
+
+class TestSingleInterface(TestCase):
+
+    def setUp(self):
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = True
+        self.client = app.test_client()
+        self.test_interfaces = Interfaces()
+        # mock_github_client = patch("webapp.integrations.logic.github_client")
+
+        self.github_interfaces_url = "".join(
```

Comment:
Is there a reason you are using `.join` instead of creating a variable with one string?

Reply:
The string is longer that the line limit of 79 so I had to break into two lines and join